features/step_definitions/navigation_steps.rb

```diff
@@ -1,25 +1,33 @@
 # frozen_string_literal: true
 
 When(/^I navigate to the home page$/) do
-  @test_site = TestSite.new
   @test_site.home.load
 end
 
 When(/^I navigate to the home page that contains expected elements$/) do
-  @test_site = TestSite.new
   @test_site.home_with_expected_elements.load
 end
 
 When(/^I navigate to the letter A page$/) do
-  @test_site = TestSite.new
   @test_site.dynamic_page.load(letter: 'a')
 end
 
 When(/^I navigate to the redirect page$/) do
-  @test_site = TestSite.new
   @test_site.redirect_page.load
 end
 
+When(/^I navigate to a page with no title$/) do
+  @test_site.no_title.load
+end
+
+When(/^I navigate to another page$/) do
```

Comment:
Vague and related to my other comment.  This isn't being used as just **any** other page, it's being used as specifically the page_with_people page, so it might make sense to just say that.